packages/echo-core/package.json

```diff
@@ -52,9 +52,10 @@
         "@rollup/plugin-babel": "^5.3.1",
         "@rollup/plugin-commonjs": "^22.0.2",
         "@rollup/plugin-node-resolve": "^13.3.0",
+        "@rollup/plugin-typescript": "^11.0.0",
         "@testing-library/react-hooks": "^8.0.1",
```

Comment:
testing-library/react-hook should be removed?

Reply:
Oh, that is correct! Removing it.